### PR TITLE
Fire an event on scale with scale details

### PR DESCRIPTION
**Pull request by Mohamed-amin**
Removed the code that it was put before for lory.js.
Simply we fire an event with the scale value, we use that on the slider to know whether to disable swiping or not.

~~Not that I added `custom-event` module which basically acts as polyfill for the browser that doesn't have `new CustomeEvent`.~~

After a second look; I think the module is not quite needed, `CustomEvent` has already good support so far: http://caniuse.com/#feat=customevent

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 import Hammer from 'hammerjs';
+import CustomEvent from 'custom-event';
```
Comment (shidhincr) [inline index.js]: Do you need to add  `custom-event` to package.json ..right ?

File: index.js [hunk 2]
```diff
@@ -64,27 +76,8 @@ class PhotoViewManager {
     return scale;
   }
 
-  imageTouchHandler(e) {
```
Comment (shidhincr) [inline index.js]: Why these got removed ? I remember these are required to fix an issue with Pan and Touchswipe

//cc @izebb

Comment (Mohamed-amin) [inline index.js]: @shidhincr Now I do that on the event, on the Slider side, we capture and disable panning.

**Comment by shidhincr** [comment]
>After a second look; I think the module is not quite needed, CustomEvent has already good support so far: http://caniuse.com/#feat=customevent

@Mohamed-amin Android 4.2-4.3 support is limited.. so we might need to use that module I guess

**Comment by Mohamed-amin** [comment]
@odino @shidhincr @izebb just added `custom-event` pollyfill
can you have a final look?

File: index.js (hunk 2 above)
Comment (izebb) [inline index.js]: @shidhincr Yes we did fix that on lory disabling touchstart and touchmove events. But I think emitting an event is a better approach with regards to delegation.

**Pull request merged**